util/rate_limiter.cc

```diff
@@ -275,52 +279,42 @@ int64_t GenericRateLimiter::CalculateRefillBytesPerPeriod(
 }
 
 Status GenericRateLimiter::Tune() {
-  const int kLowWatermarkPct = 50;
-  const int kHighWatermarkPct = 90;
-  const int kAdjustFactorPct = 5;
-  // computed rate limit will be in
-  // `[max_bytes_per_sec_ / kAllowedRangeFactor, max_bytes_per_sec_]`.
   const int kAllowedRangeFactor = 20;
+  const int kRatioLower = 15;
+  const int kRatioUpper = 50;
```

Comment:
normal write amp could be 20-30X or even higher. I think we need a large enough value for the upper bound. If this value is not useful for the algorithm, how about remove it.

Reply:
Agree to remove it, we already have max-bytes-per-sec as an upper bound.